### PR TITLE
SC1-2-2-captions-prerecorded - first incomplete draft in md

### PR DESCRIPTION
**Pull request by YetAnotherEye**
The first one :)

### CODE REVIEW

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+# SC1-2-2-captions-prerecorded
+
+## Description
+This test checks that prerecorded synchronized media (almost video) contains captions, which are a complete substitute of the audio content.
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: (almost video)? What is that supposed to say?

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+| User expertise and skills | no prior knowledge
+| User profile              | Requires vision + hearing
+## Test procedure
+
+### Selector
+Test method: [manual]
+
+All pages including elements capable of playing video.
+
+Those can be identified by including elements having an attribute value containing a media file format, such as  .mov, .rm, .ram, .asf, .swf, .dcr, .avi, .mpg, .mpeg, .mp4, .m4v, .webm, .flv, .isma
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: I think we should specify which attributes to look at. You can't just go: "look at every attribute". That's impractical to implement.

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+| Test environment          | rendered page + server connection
+| Test subject              | single web page
+| User expertise and skills | no prior knowledge
+| User profile              | Requires vision + hearing
+## Test procedure
+
+### Selector
+Test method: [manual]
+
+All pages including elements capable of playing video.
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Selectors are used within a page, not to a collection of pages. The rule is also applied to a single page, so all pages doesn't work well here. Maybe select the body if it contains something like this?

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+| Test requirement          | SC1-2-2-captions-prerecorded
+| Test mode                 | manual
+| Test environment          | rendered page + server connection
+| Test subject              | single web page
+| User expertise and skills | no prior knowledge
+| User profile              | Requires vision + hearing
+## Test procedure
+
+### Selector
+Test method: [manual]
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Should be automatic

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+- The user defines if this test case is applicable to content that is not a video but animated
+- The synchronized media is not a media alternative for another media
+- The test does not check for programmatic determinability of the media alternative or a reference such as "The description can be found in text below" from the short text alternative of the video to it.
+
+## Test properties
+| Property                  | Value
+|---------------------------|------------------------------------------|
+| Test name                 | Captions on prerecorded synchronized media
+| Test requirement          | SC1-2-2-captions-prerecorded
+| Test mode                 | manual
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: semi-automatic. The selector is automatic.

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+Check that the prerecorded video content has relevant information for the context of the web page.
+
+**User Input Question:**
+
+| Property             | Value
+|----------------------|-------
+| presented-item       | Whole page
+| requires-context     | yes
+| requires-interaction | yes
+| question             | Does the video content not contain information relevant for the context?
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Quite a difficult question. Could be simplified to:
"Is the video exclusively used as a decorative element on this page?"

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+Check that there is an alternative version available for the prerecorded video content.
+
+**User Input Question:**
+
+| Property             | Value
+|----------------------|-------
+| presented-item       | Whole page
+| requires-context     | yes
+| requires-interaction | yes
+| question             | Is there an alternative version for the video content available?
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Would prefer if we just asked if the information in the video was also available as text on the page.

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 8]
```diff
@@ -0,0 +1,171 @@
...
+|----------|-----
+| Testcase | SC1-2-2-captions-prerecorded
+| ID       | SC1-2-2-captions-prerecorded-fail1
+| Error    | Missing alternative version.
+
+
+## Step 5
+Test method: [manual]
+
+Check that the alternative version has the same information presented as the video-only content.
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: This step can be combined with the previous

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+## Background
+- [Captions (Prerecorded): Understanding SC 1.2.2](http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html)
+* [G93: Providing open (always visible) captions](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G93)
+* [G87: Providing closed captions](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G87)
+* [F8: Failure of Success Criterion 1.2.2 due to captions omitting some dialogue or important sound effects](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F8)
+* [F75: Failure of Success Criterion 1.2.2 by providing synchronized media without captions when the synchronized media presents more information than is presented on the page](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F75)
+* [F74: Failure of Success Criterion 1.2.2 and 1.2.8 due to not labeling a synchronized media alternative to text as an alternative](http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/F74)
+
+## Assumptions
+- The user defines if this test case is applicable to content that is not a video but animated
```
Comment (rdeltour) [inline drafts/SC1-2-2-captions-prerecorded.md]: 👍, can't we generally assume that basic understanding of WCAG is required for all rules?

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 10]
```diff
@@ -0,0 +1,171 @@
...
+- The synchronized media is not a media alternative for another media
```
Comment (rdeltour) [inline drafts/SC1-2-2-captions-prerecorded.md]: Can this be an assumption or does it need to be part of the rule's logic? (either in the selector or as a first step)

File: drafts/SC1-2-2-captions-prerecorded.md (hunk 2 above)
Comment (rdeltour) [inline drafts/SC1-2-2-captions-prerecorded.md]: Also, external assets might not be the only kind of synchronized media? What about for instance inline SVG animations? canvas animation? etc?

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 11]
```diff
@@ -0,0 +1,171 @@
...
+Check that the content is a video containing an audio-track.
+
+**User Input Question:**
+
+| Property             | Value
+|----------------------|-------
+| presented-item       | Media content
+| requires-context     | yes
+| requires-interaction | yes
+| question             | Does the media consist of video combined with audio?
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Alister: Maybe a good first question would be not to look at "audio" but to look specifically for dialog.

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 12]
```diff
@@ -0,0 +1,171 @@
...
+### Selector
+Test method: [manual]
+
+All pages including elements capable of playing video.
+
+Those can be identified by including elements having an attribute value containing a media file format, such as  .mov, .rm, .ram, .asf, .swf, .dcr, .avi, .mpg, .mpeg, .mp4, .m4v, .webm, .flv, .isma
+
+    XPath: //*[contains(@*,'.mov') or contains(@*,'.rm') or ... ]
+
+### Step 1
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Should this be included in the selector.

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 13]
```diff
@@ -0,0 +1,171 @@
...
+## Test procedure
+
+### Selector
+Test method: [manual]
+
+All pages including elements capable of playing video.
+
+Those can be identified by including elements having an attribute value containing a media file format, such as  .mov, .rm, .ram, .asf, .swf, .dcr, .avi, .mpg, .mpeg, .mp4, .m4v, .webm, .flv, .isma
+
+    XPath: //*[contains(@*,'.mov') or contains(@*,'.rm') or ... ]
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Alister: Look at video elements as well.

File: drafts/SC1-2-2-captions-prerecorded.md (hunk 6 above)
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Alister: Maybe talk something about "is the video meaningful in the context of the page".

File: drafts/SC1-2-2-captions-prerecorded.md (hunk 11 above)
Comment (rdeltour) [inline drafts/SC1-2-2-captions-prerecorded.md]: > Maybe a good first question would be not to look at "audio" but to look specifically for dialog.

(for the record, as I mentioned that in our call)
Not quite sure about this one: there are videos with a story, where audio contribute to the story, but with no proper dialog.

For example, see ["la linea"](https://www.youtube.com/watch?v=skb2gKR7rOk) series.

Presence of a dialog is a hint, but not a necessary criterion.

File: drafts/SC1-2-2-captions-prerecorded.md [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+
+### Step 1
+Test method: [manual]
```
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Alister + Emma, swap step 1 and step 2.

File: drafts/SC1-2-2-captions-prerecorded.md (hunk 7 above)
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Alister: "This should be 'caption' not 'alternative version'".

File: drafts/SC1-2-2-captions-prerecorded.md (hunk 8 above)
Comment (WilcoFiers) [inline drafts/SC1-2-2-captions-prerecorded.md]: Alister: "Tell users to watch the video"

**Pull request merged**